### DISCUSSION TITLE
六角龙机器人（edge浏览器翻译）是如何工作的？

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#1732] [提问] [answered]
六角龙机器人@HexDragon-Bot（edge浏览器翻译）是如何工作的？他是如何做到自动添加标签或者关闭issue的？

**Comment by WForst-Breeze** [comment, marked as answer]
#894 中龙猫给出了相应回复，解释如下：
> github 提供了 API，你可以翻翻文档……Graph API 和 Rest API（

**Comment by Deep-Dark-Forest** [comment]
@
